x-pack/plugins/security_solution/common/detection_engine/schemas/request/import_rules_schema.test.ts

```diff
@@ -1791,5 +1791,84 @@ describe('import rules schema', () => {
       };
       expect(message.schema).toEqual(expected);
     });
+
+    describe('threat_mapping', () => {
+      test('You can set a threat query, index, mapping, filters on a pre-packaged rule', () => {
```

Comment:
Nit: quick wording fix:
```suggestion
      test('You can set a threat query, index, mapping, filters on an imported rule', () => {
```